.pre-commit-config.yaml

```diff
@@ -5,5 +5,5 @@ repos:
         name: ruff
         files: '(.*\.py$|.*\.pyi$)'
         language: python
-        entry: ruff
+        entry: ruff check .
```

Comment:
I think follow the ruff pre-commit example is probably good to do :)
https://github.com/astral-sh/ruff-pre-commit
And this is how pytest use ruff
https://github.com/pytest-dev/pytest/blob/main/.pre-commit-config.yaml#L2-L7